exp/tests/test_email_templates.py

```diff
@@ -0,0 +1,243 @@
+import uuid
+
+from bs4 import BeautifulSoup
+from django.template.loader import render_to_string
+from django.test import TestCase
+from django_dynamic_fixture import G
+from parameterized import parameterized
+
+from accounts.models import User
+from studies.models import Study, StudyType
+
+
+class TestEmailTemplates(TestCase):
+    def test_notify_researcher_of_lab_permissions_html_lab_url(self):
+        context = {"lab_id": 4321}
+        html = render_to_string(
+            "emails/notify_researcher_of_lab_permissions.html", context
+        )
+        bs = BeautifulSoup(html, "html.parser")
+        self.assertEqual(bs.a["href"], "https://localhost:8000/exp/labs/4321/")
```

Comment:
Is it a problem to hard code the BASE_URL value here (https://localhost:8000/)? Do we know whether this will work in other test environments? I wonder if we need to import this from project settings, or maybe set it explicitly the way you did for `TestGetAbsoluteURLTestCase` (`@override_settings(BASE_URL="https://fakedomain.asdf/")`).

Reply:
That's valid. Gimme 30 mins and I'll push an update.